### DISCUSSION TITLE
Paid plan info on multiple domains

### DISCUSSION THREAD

**Discussion by klarity-svelte** [vercel/community#4253] [Feature Question] [answered]
### Question

Hello! Out product consists of multiple parts. Multiple SvelteKit apps + single app has multiple domains for different regions. Is that possible on a single $20 plan?

**Comment by naegelin** [comment, marked as answer]
Its possible to have multiple projects on a single pro plan.  You can also assign multiple domains to one project.

**Reply by amyegan** [reply]
In case anyone needs more detail, additional info about using custom domains can be found on the following pages:
- [Adding & Configuring a Custom Domain](https://vercel.com/docs/projects/domains/add-a-domain#)
- [Redirecting domains](https://vercel.com/docs/edge-network/redirects#)
- [Wildcard domain](https://vercel.com/docs/projects/domains/working-with-domains#wildcard-domain)
